CONTRIBUTING.rst

```diff
@@ -5,28 +5,27 @@ Contributing
 Contributions are welcome and appreciated!
 Every little bit helps, and credit will always be given.
 
-If you are new to ScanCode and want to find easy tickets to work on, check https://github.com/nexB/scancode-toolkit/labels/easy
+If you are new to ScanCode and want to find easy tickets to work on, check `issues <https://github.com/nexB/scancode-toolkit/labels/easy>`_
 
 When contributing to ScanCode (such as code, bugs, documentation, etc.) you
-agree to the Developer Certificate of Origin http://developercertificate.org/
-and the ScanCode license (see the NOTICE file).  The same approach is used
+agree to the Developer `Certificate of Origin <http://developercertificate.org/>`_
+and the ScanCode license (see the `NOTICE <https://github.com/nexB/scancode-toolkit/blob/develop/NOTICE>`_ file).  The same approach is used
 by the Linux Kernel developers and several other projects.
 
 For commits, it is best to simply add a line like this to your commit message,
 with your name and email::
 
     Signed-off-by: Jane Doe <developer@example.com>
 
-Please try to write a good commit message.
-See https://github.com/nexB/aboutcode/wiki/Writing-good-commit-messages for details.
+Please try to write a good commit message, see `this <https://github.com/nexB/aboutcode/wiki/Writing-good-commit-messages>`_ for details.
+For writing good commit messages and comments with markdown, read `this guide <https://guides.github.com/features/mastering-markdown/>`_ for once
 
 
 
 Feature requests and feedback
 =============================
 
-To send feedback or ask a question file an issue at
-https://github.com/scancode/scancode-toolkit/issues
+To send feedback or ask a question file an issue at `issues <https://github.com/scancode/scancode-toolkit/issues>`_ 
```

Comment:
Adding "at issues" to the end seems a bit redundant. How about making "file an issue" the link?

Reply:
sure